grakn-engine/src/main/java/ai/grakn/engine/backgroundtasks/distributed/KafkaLogger.java

```diff
@@ -19,15 +19,14 @@
 package ai.grakn.engine.backgroundtasks.distributed;
 
 import org.apache.commons.lang.exception.ExceptionUtils;
-import org.apache.kafka.clients.producer.KafkaProducer;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 public class KafkaLogger {
     private final static Logger LOG = LoggerFactory.getLogger(KafkaLogger.class);
     private final LogLevel logLevel;
     private static KafkaLogger instance = null;
-    private KafkaProducer<String, String> producer;
+//    private KafkaProducer<String, String> producer;
```

Comment:
This one is commented because of existing commented code in this class (that used this field). We should remove all of it though!